dali/operators/reader/loader/video/video_test_base.h

```diff
@@ -38,10 +38,21 @@ class VideoTestBase : public ::testing::Test {
 
   void CompareFrames(const uint8_t *frame, const uint8_t *gt, int size, int eps = 10);
 
+  void CompareFramesAvgError(const uint8_t *frame, const uint8_t *gt, int size, double eps = 1.0);
+
   uint8_t *GetCfrFrame(int video_id, int frame_id) { return cfr_frames_[video_id][frame_id].data; }
 
   uint8_t *GetVfrFrame(int video_id, int frame_id) { return vfr_frames_[video_id][frame_id].data; }
 
+  void SaveFrame(
```

Comment:
Can you add a docstring to describe how to use it?
What is the meaning of frame_id, sample_id, and batch_id?

Reply:
Done